feat(fast-data): migration guide from current architecture to event-driven (#782)
mia-platform/documentation: docs/fast_data/faq/architecture_migration.md

---
id: architecture_migration
title: Architecture migration
sidebar_label: Architecture migration
---

> As you know, the current architecture of the Fast Data is composed of two main services, the [Real-Time Updater](/fast_data/realtime_updater.md) and the [Single View Creator](/fast_data/single_view_creator.md). We are introducing two new additions to our core services, the [Projection Storer](/fast_data/realtime_updater.md) and the [Single View Trigger Generator](/fast_data/single_view_trigger_generator.md) used on what we called the [event-driven architecture](/fast_data/architecture.md#event-driven-architecture) of the Fast Data.

Reviewer: ```suggestion
The [standard architecture](/fast_data/architecture.md#standard-architecture) of Fast Data is composed of two main services, the [Real-Time Updater](/fast_data/realtime_updater.md) and the [Single View Creator](/fast_data/single_view_creator.md). Two new additions to our core services, the [Projection Storer](/fast_data/realtime_updater.md) and the [Single View Trigger Generator](/fast_data/single_view_trigger_generator.md) have to be used on what we called the [event-driven architecture](/fast_data/architecture.md#event-driven-architecture) of the Fast Data.
```

Author: Non mi fila liscio completamente ne il prima ne il dopo. Provo a riformulare aggiugendo il link consigliato

Author: fatto, dimmi cosa ne pensi
